python/ql/test/3/query-tests/Statements/iter/supress_uses_from_tests.py

```diff
@@ -0,0 +1,19 @@
+# This example illustrates that all valid results are not useful.
```

Comment:
I interpret this as saying `forall(Result r | r.valid() implies not r.useful())`, which sounds a bit odd to me.

Did you intend to say `not forall(Result r | r.valid() implies r.useful())`?
(i.e. `not all valid results are useful`?)

Reply:
:smile: yes